src/sinol_make/util.py

```diff
@@ -79,7 +79,8 @@ def install_oiejq():
         raise Exception('Couldn\'t download oiejq (https://oij.edu.pl/zawodnik/srodowisko/oiejq.tar.gz couldn\'t connect)')
     if request.status_code != 200:
         raise Exception('Couldn\'t download oiejq (https://oij.edu.pl/zawodnik/srodowisko/oiejq.tar.gz returned status code: ' + str(request.status_code) + ')')
-    open('/tmp/oiejq.tar.gz', 'wb').write(request.content)
+    with open('/tmp/oiejq.tar.gz', 'wb') as oiejq_file:
+        oiejq_file.write(request.content)
```

Comment:
Sometimes you're using a cache directory in the project, sometimes a `/tmp` directory. Let's be consistent.

Reply:
I think it's possible in Python to e.g. create a directory that gets removed after the code leaves a 'with' block. Might be helpful here, but I haven't put much thought into it.

Reply:
I changed it

Reply:
We're still inconsistent -- here we use a `/tmp` directory, in other places we use the cache directory in the project.

Reply:
I think that oiejq installation should be in the temp directory, as there is no guarantee that the current directory is a package, and I thought the cache dir is for stuff the package creator might want to look into.

Reply:
Sounds reasonable. You can write an explanation in a comment.

Reply:
In fact, anything that is different than the usual style should have a comment, as probably I'll nevertheless ask about it during code review.

Reply:
I added the comment.